cloudwash/utils.py

```diff
@@ -81,3 +84,11 @@ def gce_zones() -> list:
     _zones_combo = {**_bcds, **_abcfs, **_abcs}
     zones = [f'{loc}-{zone}' for loc, zones in _zones_combo.items() for zone in zones]
     return zones
+
+def discovery_timestamp(resource):
```

Comment:
This should also be moved to `wrapanapi` !

Reply:
@jyejare Ack on this review, will be resolved in #134. Thanks